### ISSUE TITLE
Show the video to the Callee before they answer (Knock Knock)

### ISSUE THREAD

**Issue by jaller94** [nextcloud/spreed#26]
Offer the caller the optional feature to transmit their video before the callee answers the call.

A live preview would be presented to the callee who then decides to answer or to decline the call.

This feature got popular through the video chatting app Google Duo:
https://duo.google.com/

**Comment by jancborchardt** [comment]
Yes, this is important to see if you look ok and also if the video works at all.
https://talky.io also gives you feedback on if your microphone works, very helpful.

**Comment by fancycode** [comment]
The "knock knock" feature works the other way around. The person being called can see a live video of the caller before picking up the call.
